src/hdf5_time_series_storage.jl

```diff
@@ -126,8 +126,10 @@ function get_data_type(ts::TimeSeriesData)
         return "POLYNOMIAL"
     elseif data_type == PWL
         return "PWL"
+    elseif data_type <: Integer
```

Comment:
We still have a problem converting SingleTimeSeries with integers to floats.  This is a workaround.

Reply:
Doesn't this happen automatically?

Reply:
Nope.  CONSTANT is now only Float64

Reply:
We should add a comment on why this is here then.